### PR TITLE
[VAS-541] feat: Implement the APIs needed in the create bundle page

### PR DESCRIPTION
**Pull request by svariant**
#### List of Changes

- Implemented `getPaymentTypes`, `getTouchpoints`, `getTaxonomies`, `getBrokerDelegation` & `createBundle` APIs
- Added new Autocomplete for brokers' delegations
- Changed date validation
- Cleaned and refactored code

#### Motivation and Context

[Issue VAS-541](https://pagopa.atlassian.net/browse/VAS-541?atlOrigin=eyJpIjoiMzQxZTAxMDFlM2JmNDkxZWIzMmVmZjNiZDhmZTg5ZjkiLCJwIjoiaiJ9)

#### How Has This Been Tested?

Tests will be defined in the future 

#### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/pages/channels/channelPSPList/ChannelPSPTable.tsx [hunk 1]
```diff
@@ -157,7 +157,7 @@ export default function ChannelPSPTable({ setAlertMessage, pspNameFilter }: Chan
                   pspListPage.page_info?.total_pages && pspListPage.page_info?.total_pages > 1 ? (
                     <Pagination
                       color="primary"
-                      count={pspListPage.page_info?.total_pages ?? 0}
+                      count={pspListPage?.page_info?.total_pages ?? 1}
```
Comment (jacopocarlini) [inline src/pages/channels/channelPSPList/ChannelPSPTable.tsx:160]: why `1`?

File: src/services/taxonomyService.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
+// import { BackofficeApi } from '../api/BackofficeClient';
+import { toError } from 'fp-ts/lib/Either';
+import {storageTokenOps} from '@pagopa/selfcare-common-frontend/utils/storage';
+import { ENV } from '../utils/env';
+import { Taxonomies } from '../api/generated/portal/Taxonomies';
+import { getTaxonomies as getTaxonomiesMock } from './__mocks__/taxonomyService';
+
+
+
+// TODO FIX GENERATED CLIENT AND RE-INTRODUCE BACKOFFICE METHOD
```
Comment (jacopocarlini) [inline src/services/taxonomyService.ts:10]: 👀

File: src/pages/channels/channelPSPList/ChannelPSPTable.tsx (hunk 1 above)
Comment (svariant) [reply src/pages/channels/channelPSPList/ChannelPSPTable.tsx:160]: If the totalPages is undefined/null the default value should be 1 for the pages, with 0 the pagination is broken and doesn't show any pages (even if there are elements for 1 page)
Just a precaution

File: src/services/taxonomyService.ts (hunk 2 above)
Comment (svariant) [reply src/services/taxonomyService.ts:10]: The service will be reworked with the new APIs for the taxonomies (that have yet to be defined)

**Pull request merged**